### DISCUSSION TITLE
VictorOps module to support templated message

### DISCUSSION THREAD

**Discussion by wengelbrecht-grafana** [grafana/grafana#46850] [Alerting]
Grafana's implementation of VictorOps integration does not allow you to set a custom message template, as you can with some of the other alert integrations.

It would be great if this can be added to Grafana. 
The VictorOps document for _state_message_ is documented here: https://help.victorops.com/knowledge-base/rest-endpoint-integration-guide/

**Comment by vivekmalhotragithub** [comment]
The slack contact point config allows templating like
<img width="960" alt="Screenshot 2022-03-22 at 17 02 34" src="https://user-images.githubusercontent.com/13301860/159712943-aec6962c-60f6-45ef-b89b-99cf4a2ba522.png">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
